layers/meta-balena-jetson/recipes-kernel/linux/linux-tegra_%.bbappend

```diff
@@ -34,7 +34,7 @@ SRC_URI_append_jetson-tx2 = " \
 "
 
 SRC_URI_append_jetson-nano = " \
-    file://tegra210-p3448-0002-p3449-0000-b00.dtb \
+    file://tegra210-p3448-0000-p3449-0000-a02.dtb \
```

Comment:
This seems to override the default device tree for jetson-nano and swap it to the JN30B one. Think it's best to only add it for JN30B,  with its original name, maybe append a JN30B_JP4.2.2 to it.. This name will be specified in tegra210-flash, as mentioned in my other comment. Should make it easier to maintain in the future. Thanks

Reply:
Makes sense, I've renamed the .dtb file name to `tegra210-p3448-0002-p3449-0000-b00-jn30b-JP4.2.2.dtb`